Makefile

```diff
@@ -234,6 +234,30 @@ deploy-test: _verify_minikube generate-keys
 	export ENABLE_AUTH="True" && export DUMMY_IGNITION="True" && \
 	$(MAKE) _update-minikube deploy-wiremock deploy-all
 
+generate-onprem-environment:
+	echo "POSTGRESQL_DATABASE=installer" > onprem-environment
```

Comment:
It seems that there are only 2 variables in this file.
Can we make it a permanent file and update its variables using a `sed` command?